app/services/letter_sending_service.rb

```diff
@@ -73,6 +75,10 @@ def address
     # split on commas unless preceded by digits (i.e. house numbers)
     address_lines = neighbour.address.split(/(?<!\d), */).compact
     address_lines.insert(0, "The Occupier")
+
+    # GOV.UK Notify expects at least 3 lines for the address
+    raise AddressValidationError, "The address: '#{neighbour.address}' is invalid. It must be at least 3 lines." if address_lines.length < 3
```

Comment:
Yeah good point

Reply:
We also can't say _three_ lines, I think, because that includes lines that we insert, i.e., "The Occupier". So in fact it's two lines, one of which must be the postcode. So is the actual problem that the user isn't providing a postcode?

Reply:
I guess it's actually checking we have at least 3 comma separated values but as you say we insert "The Occupier" so it's just two comma separated values that needs to be provided. 
So even suggesting a postcode is added could raise an error if it is given as "60 Commercial Street E1 6PT" etc

Reply:
Yeah, I think I wrote the code to split on commas with the assumption that it would always be coming from the search. I don't think it's possible to handle an arbitrary string in a robust enough way.

We could tell the user they should make sure the lines of the address are separated by commas?